### PR TITLE
First part of the app is done

### PR DESCRIPTION
**Pull request by gcarolina**

### CODE REVIEW

File: RegistrationApp/RegistrationApp/Controllers/CreateAnAccountViewController.swift [hunk 1]
```diff
@@ -0,0 +1,153 @@
+//  CreateAnAccountViewController.swift
+//  RegistrationApp
+//  Created by Carolina on 18.11.22.
+
+import UIKit
+
+class CreateAnAccountViewController: UIViewController {
```
Comment (progmev) [inline RegistrationApp/RegistrationApp/Controllers/CreateAnAccountViewController.swift:7]: final

File: RegistrationApp/RegistrationApp/Controllers/CreateAnAccountViewController.swift [hunk 2]
```diff
@@ -0,0 +1,153 @@
...
+    @IBOutlet private weak var errorPasswordLbl: UILabel!
+    /// password indicators
+    @IBOutlet var strongPassIndicatorsViews: [UIView]!
+    /// confirm password
+    @IBOutlet private weak var confPasswordTF: UITextField!
+    @IBOutlet private weak var confPasswordLbl: UILabel!
+    /// continue button (чтобы enable/disable использовать)
+    @IBOutlet private weak var continueBtn: UIButton!
+    /// scrollView (чтобы приподнимать scrollView при открытой клавиатуре)
+    @IBOutlet private weak var scrollView: UIScrollView!
```
Comment (progmev) [inline RegistrationApp/RegistrationApp/Controllers/CreateAnAccountViewController.swift:27]: private weak

File: RegistrationApp/RegistrationApp/Controllers/CreateAnAccountViewController.swift [hunk 3]
```diff
@@ -0,0 +1,153 @@
...
+    // MARK: - Navigation
+    // переход от CreateVC к SecretCodeVC
+    @IBAction func continueAction() {
+        let storyboard = UIStoryboard(name: "SecretCodeStoryboard", bundle: nil)
+        if let secretCodeVC = storyboard.instantiateViewController(withIdentifier: "SecretCodeViewController") as? SecretCodeViewController {
+            navigationController?.pushViewController(secretCodeVC, animated: true)
+            secretCodeVC.email = emailTF.text
+            secretCodeVC.nameOfPearson = nameTF.text
+        }
+    }
```
Comment (progmev) [inline RegistrationApp/RegistrationApp/Controllers/CreateAnAccountViewController.swift:89]: private

**Review by progmev — APPROVED** [review]
+1

File: RegistrationApp/RegistrationApp/Controllers/CreateAnAccountViewController.swift (hunk 1 above)
Comment (gcarolina) [reply RegistrationApp/RegistrationApp/Controllers/CreateAnAccountViewController.swift:7]: fixed

File: RegistrationApp/RegistrationApp/Controllers/CreateAnAccountViewController.swift (hunk 2 above)
Comment (gcarolina) [reply RegistrationApp/RegistrationApp/Controllers/CreateAnAccountViewController.swift:27]: fixed

**Comment by gcarolina** [comment]
Project is done

**Pull request merged**